rustls-libssl/src/lib.rs

```diff
@@ -1,5 +1,6 @@
-use core::ffi::{c_int, c_uint, c_void, CStr};
+use core::ffi::{c_char, c_int, c_uint, c_void, CStr};
 use core::{mem, ptr};
+use std::ffi::CString;
```

Comment:
nit: i think the `CString` import should be grouped with the other `core::ffi` types on L1.

Reply:
Unfortunately `CString` doesn't exist in `core::ffi`, only `std` (or `alloc`)

Reply:
Oops, totally missed that it wasn't the same top-level namespace.